### ISSUE TITLE
Vídeo para o canal!

### ISSUE THREAD

**Issue by LucasMedeiros7** [akitaonrails/rinhabackend-rails-api#1]
Poderia fazer uma video no canal sobre a Rinha e como foi desenvolver uma solução!

**Comment by diegolikescode** [comment]
Concordo 300%, mas ao mesmo tempo vale dizer uma coisa: olhando o código e as msgs de commit é possível ter uma boa noção de como foi o desenvolvimento e a linha de raciocínio :pray:

**Comment by gilmarodp** [comment]
Vídeo sobre a Rinha!
